src/App.js

```diff
@@ -2,26 +2,53 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Preloader from './components/Preloader';
 import './App.css';
-import Footer from './components/Footer';
-import E404 from './components/E404';
+
 const Home = React.lazy(() => {
   return new Promise(resolve => {
     setTimeout(() => resolve(import('./components/home')), 4000);
   });
 });
 
+const Footer = React.lazy(() => {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(import('./components/Footer')), 5000);
+  });
+});
+
+const E404 = React.lazy(() => {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(import('./components/E404')), 3000);
+  });
+});
+
 function App() {
   return (
     <>
       <Router>
         <React.Suspense fallback={<Preloader />}>
           <Switch>
             <Route exact path='/' component={Home} />
+
             <Route path='*' component={E404} />
           </Switch>
         </React.Suspense>
       </Router>
-      <Footer />
+      <React.Suspense
+        fallback={
```

Comment:
What this fallback is for?
Is it for the time period until the lazy components are shown?

Reply:
@akshay1027 I guess you missed the question

Reply:
Oh okay so you asked about fallback itself is it, yes
It is for the time period until the lazy components are shown